doc/api/util.md

```diff
@@ -1053,6 +1056,8 @@ changes:
       times. If `true`, all values will be collected in an array. If
       `false`, values for the option are last-wins. **Default:** `false`.
     * `short` {string} A single character alias for the option.
+    * `default` {string | boolean | string[] | boolean[]} The default option
+      value when it is not set by args.
```

Comment:
Document that this must match the arg type?

Reply:
Done, adding the `multiple` case too